peepingtom/functions.py

```diff
@@ -1,36 +1,41 @@
 from pathlib import Path
-from inspect import signature
 
 from .peeper import Peeper
 from .datablocks import DataBlock
-from .utils import listify
+from .utils import listify, inherit_signature
 from .io_ import read
 
 
-def peep(obj, *args, **kwargs):
+def which_isinstance(iterable, type):
```

Comment:
I feel like the name of this isn't super descrpitive... isn't this what filter is for?
i.e. `filter(lambda x: isinstance(x, type), iterable)`

could also be faster as a list comprehension?
`[item for item in iterable if isinstance(item, type)]`

Reply:
Oef, you're totally right, but that code is actually unused xD I went trhough several iterations of that, end it ended up being exactly like filter; then, I stopped using it xD

Reply:
Removed.